packages/theming/default-theme/src/defaultTheme.ts

```diff
@@ -88,6 +88,65 @@ function _defaultTypography(): Typography {
   return defaultsDict;
 }
 
+export function defaultEffects(): Effects {
+  const effects =
+    Platform.OS === 'android' || Platform.OS === 'ios'
+      ? {
+          borderRadiusSmall: 2,
+          borderRadiusMedium: 4,
+          borderRadiusLarge: 8,
+          borderRadiusXLarge: 12,
+        }
+      : {
+          borderRadiusSmall: 2,
+          borderRadiusMedium: 4,
+          borderRadiusLarge: 6,
+          borderRadiusXLarge: 8,
+        };
```

Comment:
Are these coming from Figma

Reply:
yea I hand copied them.  Mobile devices get one set, Large screen gets another.  I did skirt over the fact that web has separate values for mobile than the native mobile values, but react-native-web isn't really a first class platform for us right now